analytics_data_api/v0/apps.py

```diff
@@ -0,0 +1,13 @@
+from django.apps import AppConfig
+from django.conf import settings
+from elasticsearch_dsl import connections
+
+
+class ApiAppConfig(AppConfig):
+
+    name = 'analytics_data_api.v0'
+
+    def ready(self):
+        super(ApiAppConfig, self).ready()
+        if settings.ELASTICSEARCH_LEARNERS_HOST:
+            connections.connections.create_connection(hosts=[settings.ELASTICSEARCH_LEARNERS_HOST])
```

Comment:
[This page](https://docs.djangoproject.com/en/1.9/ref/applications/#django.apps.AppConfig.ready) warns against interacting with the database in the `ready` method. Apparently it's called every time a management command is invoked. Since we're not querying Elasticsearch here, it's probably not an issue, right? cc @mulby